app/models/crop.rb

```diff
@@ -28,6 +24,22 @@ class Crop
   field :row_spacing, type: Integer
   field :height, type: Integer
 
+  # Naming. This is tricky.
+  # See https://github.com/openfarmcc/OpenFarm/issues/641
+  field :binomial_name
+
+  field :genus
+  field :species
+
+  field :cultivar_name
+
+  field :name
+  field :common_names, type: Array
+  validates_presence_of :name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.